composer/models/classify_mnist/model.py

```diff
@@ -8,8 +8,8 @@
 import torch.nn as nn
 from torch.nn import functional as F
 
-from composer.models.base import ComposerClassifier
-from composer.models.model_hparams import Initializer
+from composer.models.base_hparams import Initializer
+from composer.models.templates import ComposerClassifier
```

Comment:
With the benchmarks folder refactor, would there be implemented models inside the `composer.models` folder? If not, do we need to subfolder the base classes?

If we did go with the subfolder approach, not a huge fan of `templates`, as that implies to me that it is something that should be copied, pasted, and modified, rather than inherited. Thoughts about `architectures`? Open to other ideas too!

Reply:
Yeah I'm not in love with the name either. These composermodels are actually everything except an architecture. They're forward pass logic + logging and they can be passed an architecture to train.

Reply:
They're task-specific (`ComposerClassifier`, `ComposerDetector`) convenience classes, so maybe that's a good approach to naming the class. Perhaps something like `task_wrapper` or `model_task`.